.gitignore

```diff
@@ -12,3 +12,5 @@ docs/man
 npm-debug.log
 .project
 test/memory.json
+intl/*
+!intl/en
```

Comment:
Let's not bikeshed on changes like this. It'll work as long as it's listed in the file somewhere.